tools/requirements.txt

```diff
@@ -4,5 +4,6 @@ python_speech_features>=0.6
 setuptools>=38.5.1
 joblib==0.11.0
 librosa>=0.6.1
+soundfile>=0.10.2
```

Comment:
I'm not super happy about using this library, as it requires `libsndfile`, and it is not so common. For now, it is OK, but we may need to find other libraries.